docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/dataset.md

```diff
@@ -26,15 +26,7 @@ title: Starwhale Dataset-数据集
 - **制品存储**：Standalone Instance能存储本地构建或分发的swds系列文件，Cloud Instance使用对象存储提供集中式的swds制品存储。
 - **Starwhale无缝集成**：`Starwhale Dataset` 能使用 `Starwhale Runtime` 构建的运行环境构建数据集。`Starwhale Evaluation` 和 `Starwhale Model` 直接通过 `--dataset` 参数指定数据集，就能完成自动数据加载，便于进行推理、模型评测等环境。
 
-### 1.3 数据集类型
-
-Starwhale Dataset 对数据集格式的定义，根据实际使用场景，有三种分类：
```

Comment:
why remove those docs? the three use cases are the starwhale dataset core scenario. We must handle `swds-bin` , `user-raw` and `remote-link` input data type.

Reply:
> why remove those docs? the three use cases are the starwhale dataset core scenario. We must handle `swds-bin` , `user-raw` and `remote-link` input data types.

`swds-bin` , `user-raw` and `remote-link` are starwhale legacy implemention specific. I think users don't care what the type really is. More oftenlly, the data in dataset is hybrid. User's could have bytes/raw data/link  in the dataset simultaneously

Reply:
chapter 5.1 is added